components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/JDBCPermissionBasedInternalScopeValidator.java

```diff
@@ -129,18 +138,29 @@
         return scopesToRespond.toArray(new String[0]);
     }
 
-    private String[] getRequestedScopes(String[] scopes) {
+    /**
+     * Execute Internal Scope Validation.
+     * @param requestedScopes Array of scopes that needs to be validated.
+     * @param authenticatedUser
+     * @param clientId
```

Comment:
Shall we add param descriptions here?

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1766/commits/41a1e41fd118816114a609736bf4363ae1ea266c